package.json

```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "lines-and-columns": "^1.1.6",
-    "tslib": "^1.9.3"
+    "tslib": "^2.0.3"
```

Comment:
Ref: https://github.com/microsoft/tslib/releases/tag/2.0.0

It does not seem to be safe to use `tslib@2.x` with `typescript@3.7.5`, we should probably update `typescript` as well.